processor/src/test/java/org/mapstruct/ap/test/bugs/_2983/Issue2983Mapper.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._2983;
+
+import org.mapstruct.AnnotateWith;
+import org.mapstruct.MapMapping;
+import org.mapstruct.Mapper;
+
+import org.mapstruct.ValueMapping;
+import org.mapstruct.ValueMappings;
+import org.mapstruct.ap.test.annotatewith.CustomAnnotation;
+import org.mapstruct.ap.test.annotatewith.CustomMethodOnlyAnnotation;
+import org.mapstruct.ap.test.value.ExternalOrderType;
+import org.mapstruct.ap.test.value.OrderType;
+import org.mapstruct.factory.Mappers;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * @author orange add
+ */
+@Mapper
+public interface Issue2983Mapper {
+
+    Issue2983Mapper INSTANCE = Mappers.getMapper( Issue2983Mapper.class );
+
+    @AnnotateWith(Deprecated.class)
```

Comment:
I know that we are using this example in our original tests, but we'll change that. Let's use some of the annotations that we already have in the package I mentioned earlier.